whd/dispatcher.py

```diff
@@ -204,3 +213,22 @@ def is_up_to_date(resource, resource_versions):
             retries=retries,
             sleep_seconds=sleep_seconds*1.2,
         )
+
+    def log_outdated_resources(self, outdated_resources):
+        whd_config = self.cfg_set.webhook_dispatcher_deployment()
```

Comment:
would it maybe make sense to mv the retrieval of elastic-client to a method (or maybe a field)?